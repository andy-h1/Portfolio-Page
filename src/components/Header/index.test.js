// MemoryRouter === BrowserRouter but for testing
import { MemoryRouter } from "react-router-dom";
// a testing version of rendering
import { render } from "@testing-library/react";
// your actual component to render
import { Header } from ".";

describe("Header component", () => {
  it("should render", () => {
    // render the component
    const { container } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );

    // checking it looks as expected
    expect(container.firstChild).toMatchSnapshot();
  });
});
