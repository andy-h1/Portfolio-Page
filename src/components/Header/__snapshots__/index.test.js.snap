// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
<header
  class="sc-AxjAm eulKPO"
>
  <h1
    class="sc-AxirZ hJCReq"
  >
    Andy Huynh
  </h1>
  <div
    class="sc-AxiKw cUMxlS"
  >
    <a
      aria-current="page"
      class="sc-AxhCb biVdMa active"
      href="/"
    >
      home
    </a>
    <a
      class="sc-AxhCb biVdMa"
      href="/cv"
    >
      cv
    </a>
    <a
      class="sc-AxhUy famNCz"
      href="https://github.com/andy-h1"
      target="_blank"
    >
      <img
        alt="github"
        class="sc-AxgMl dewwFw"
        src="github.png"
      />
    </a>
    <a
      class="sc-AxhUy famNCz"
      href="https://twitter.com/andyh_1_"
      target="_blank"
    >
      <img
        alt="twitter"
        class="sc-AxgMl dewwFw"
        src="twitter.svg"
      />
    </a>
    <a
      class="sc-AxhUy famNCz"
      href="https://www.linkedin.com/in/andyhuynh1/"
      target="_blank"
    >
      <img
        alt="linkedin"
        class="sc-AxgMl dewwFw"
        src="linkedin.png"
      />
    </a>
  </div>
</header>
`;
